lib/gui/components/drive-selector/templates/drive-selector-modal.tpl.html

```diff
@@ -50,7 +50,17 @@ <h4 class="list-group-item-heading">{{ drive.description }} - {{ drive.size | gi
 </div>
 
 <div class="modal-footer">
-  <button class="button button-primary button-block"
+  <div class="modal-footer-warning"
+    ng-show="modal.state.hasDrive() && modal.state.isSystemDrive(modal.state.getDrive())">
```

Comment:
This expression (`modal.state.hasDrive() && modal.state.isSystemDrive(modal.state.getDrive())`) is also used in the `ng-show` from the button icon, and to decide whether the button will be red or not. I wonder if we can extract this to the model controller, as something like `.showFooterWarning()` and potentially unit test it as well.